crates/ruma-api/tests/conversions.rs

```diff
@@ -52,7 +52,11 @@ fn request_serde() {
 
     let http_req = req
         .clone()
-        .try_into_http_request::<Vec<u8>>("https://homeserver.tld", SendAccessToken::None)
+        .try_into_http_request::<Vec<u8>>(
+            "https://homeserver.tld",
+            SendAccessToken::None,
+            &[MatrixVersion::V1_0],
```

Comment:
Once v1.1 paths are in place with https://github.com/ruma/ruma/pull/866, i'm probably going to convert these to use V1_1 and v3 paths instead.